Maestro/subscriptions.json

```diff
@@ -83,6 +89,16 @@
           "maestroDelay": "00:05:00"
         }
       ]
+    },
+    {
+      "path": "https://github.com/dotnet/versions/blob/master/build-info/dotnet/cli/rel/1.0.0/Latest_Packages.txt",
```

Comment:
You probably need @sokket's change and respond to the "build.version" file:  https://github.com/dotnet/cli/pull/3270/files#diff-09865b5728dc444ab9bdd51db40314efR112